### PR TITLE
More comfortable variable num players

### PR DESCRIPTION
**Pull request by rocketkai**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,4 +1,390 @@
...
+
+// Set an array that will contains details about each player (i.e. an array of objects)
+var playerProfiles = [];
+
+// Track the current round's winner
+var roundWinner = null;
+
+/*====================================
+==============HELPER FUNCTIONS========
+======================================*/
```
Comment (kai-rocket) [inline script.js:39]: See comment in other PR on sections. I don't think these sections are necessary.

File: script.js [hunk 2]
```diff
@@ -1,4 +1,390 @@
...
+ * Sort an array such that it can order any number of elements
+ * @param {Array} anArray
+ * @returns {Array} anArray sorted in ascending order
+ */
+var sortAnArray = function (anArray) {
+  // get the length of the given array
+  var length = anArray.length;
+
+  //Use bubble sort to sort a given array
+  // More info on bubble sort: https://www.geeksforgeeks.org/bubble-sort/
```
Comment (kai-rocket) [inline script.js:84]: Nice easter egg!

File: script.js [hunk 3]
```diff
@@ -1,4 +1,390 @@
...
+/**
+ * Sort an array such that it can order any number of elements
+ * @param {Array} anArray
+ * @returns {Array} anArray sorted in ascending order
+ */
+var sortAnArray = function (anArray) {
+  // get the length of the given array
+  var length = anArray.length;
+
+  //Use bubble sort to sort a given array
```
Comment (kai-rocket) [inline script.js:83]: Please have consistent spacing after `//`!

File: script.js [hunk 4]
```diff
@@ -1,4 +1,390 @@
...
+  playerProfiles[currPlayer].diceNum = playerNum;
+  // Return generated player num to parent function
+  return playerNum;
+};
+
+/**
+ * Update the global variable that tracks  players' current round number
+ * @param {number} firstNumeralIndex
+ */
+updateCurrentRoundRoll = function (playerNum) {
```
Comment (kai-rocket) [inline script.js:139]: Missing `var`

**Review by rocketkai — COMMENTED** [review]
Generally clean and nice. Great work @jeremylim91! CC @awongh

**Pull request closed without merging**